recipes/ri/meta.yaml

```diff
@@ -0,0 +1,35 @@
+{% set version = "3.6" %}
+
+package:
+  name: ri
+  version: {{ version }}
+
+source:
+  url: https://github.com/InfOmics/RI/archive/{{ version }}.zip
+  sha256: 5168d47f3882a526be9276b57684b8c2d234ede80e42865f0039e6a5a432c254
+  patches:
+    - makefile.patch
+    
+build:
+  number: 0
+  skip: True  # [win]
+
+requirements:
+  build:
+    - make
+    - {{ compiler('cxx') }}
+
+test:
+  commands:
+    - which ri36
```

Comment:
We usually use `test -f ${PREFIX}/bin/ri36` here

Reply:
Done